Dockerfile

```diff
@@ -0,0 +1,82 @@
+# FROM debian:buster-slim as dqlite-lib-builder 
+FROM ubuntu as dqlite-lib-builder 
+ARG DEBIAN_FRONTEND="noninteractive"
+ENV TZ=Europe/London
+ENV LD_LIBRARY_PATH=/usr/local/lib
+ENV GOROOT=/usr/local/go
+ENV GOPATH=/go
+ENV PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+
+RUN apt-get update && apt-get install -y git build-essential dh-autoreconf pkg-config libuv1-dev tcl8.6 wget
+
+WORKDIR /opt
+
+RUN git clone --depth 100 https://github.com/canonical/sqlite.git
+
+WORKDIR /opt/sqlite
+
+RUN ./configure --enable-replication && make && make install
+
```

Comment:
These steps to build SQLite can be dropped. You can just install the `libsqlite3-dev` package from Debian or Ubuntu (no particular version required) and things will work.

We merged #264 today, and that gets rid of the patched version of SQLite, plain upstream SQLite is now sufficient.

Reply:
Fixed in https://github.com/canonical/dqlite/pull/267/commits/8256b2d567559a7cc5a84a58303ffd6c81ad4a8c

Reply:
I spoke too fast. Tried again in https://github.com/canonical/dqlite/pull/267/commits/42f0085210c706a8f8eee2872e5d744e7e54c5e3

Looking a little better:

```
REPOSITORY        TAG                       SIZE
dqlite              ubuntu                    103MB

```